### PR TITLE
Deprecate export of module handlers via `EXPORTED_RUNTIME_METHODS`

### PR DESCRIPTION
**Pull request by kleisauke**
`print` / `printErr` are handlers that can be overridden by the user on the incoming module (just like `onAbort`) and not runtime elements that can be exported. Deprecate the export of `print` / `printErr` via `EXPORTED_RUNTIME_METHODS` in favor of `out` / `err`.

### CODE REVIEW

File: src/modules.js [hunk 1]
```diff
@@ -311,6 +311,10 @@ function addMissingLibraryStubs() {
 function exportRuntime() {
   const EXPORTED_RUNTIME_METHODS_SET = new Set(EXPORTED_RUNTIME_METHODS);
 
+  const legacyRuntimeElements = new Map();
+  legacyRuntimeElements.set('print', 'out');
+  legacyRuntimeElements.set('printErr', 'err');
```
Comment (sbc100) [inline src/modules.js:316]: Does JS not have a literal syntax for this?

**Review by sbc100 — APPROVED** [review]

File: src/modules.js (hunk 1 above)
Comment (kleisauke) [reply src/modules.js]: TIL, https://stackoverflow.com/a/35962243. See commit f6c2f803c2085ab474af5c9664b6078c12a4000c.

**Review by kripken — APPROVED** [review]
Please add a note to the changelog.

**Comment by kleisauke** [comment]
Sorry for the delay, I just added a note to the changelog.

**Pull request merged**